instaclustr/resource_cluster.go

```diff
@@ -1114,7 +1115,7 @@ func doClusterResize(client APIClientInterface, clusterID string, d resourceData
 	}
 	switch cluster.BundleType {
 	case "APACHE_CASSANDRA":
-		if hasCassandraSizeChanges(d) {
+		if hasSimpleNodeSizeChanges(d) {
```

Comment:
**[SUGGESTIONS]** There seems like a lot of repetition of the if else for each bundle. Is there a way to refactor it? 
`if(hasSimpleNodeSizeChanges(d)){return bundleClusterResize}`
The function `bundleClusterResize` takes the bundle type as parameter and do the switch check there.
Not quite sure the best way to go about it but that's one of the ideas^^